extension/src/experiments/commands/index.ts

```diff
@@ -48,7 +48,7 @@ export const getRenameExperimentCommand =
 
 const promptToAddStudioToken = async () => {
   const response = await Toast.askShowOrCloseOrNever(
-    `Experiments can be automatically shared to [Studio](${STUDIO_URL}) by setting the studio.token in your config.`
+    `Experiments can be automatically shared to [Studio](${DEFAULT_STUDIO_URL}) by setting the studio.token in your config.`
```

Comment:
We only use the url found in the config for fetch requests to studio (aka requesting a token or studio experiment data). 

For links in the webview or Toasts, we just use the default one.

Reply:
why?

Reply:
Do you mean for "generic" links when the user doesn't have an account or when they don't have a token setup? I thought that "links in the webview" meant "shareable experiment links".

Reply:
> Do you mean for "generic" links when the user doesn't have an account or when they don't have a token setup

Yes, apologies, I didn't explain that very well 🤦‍♀️